src/oops/SOLID/lsp/stack/before/StackWrong.java

```diff
@@ -36,7 +37,5 @@ public static void main(String[] args) {
 		System.out.println(st.top());
 		st.pop();
 		System.out.println(st.top());
-		st.clear();
```

Comment:
Replace clear with pop instead of removing it.

Reply:
get it. will change.